spec/features/delius_import_job_feature_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+require 'support/lib/mock_imap'
+require 'support/lib/mock_mail'
+require 'delius_import_job'
+
+feature "delius import feature" do
+  let(:stub_auth_host) { Rails.configuration.nomis_oauth_host }
+  let(:stub_api_host) { "#{stub_auth_host}/elite2api/api" }
+  let(:offender_no) { "GCA2H2A" }
+  let(:prison) { "LEI" }
+  let(:booking_id) { 754_207 }
+  let(:offenders) {
+    [{
+      "bookingId": booking_id,
+      "offenderNo": offender_no,
+      "dateOfBirth": "1985-03-19",
+      "agencyId": prison
+    }]
+  }
+  let(:bookings) {
+    [{  "bookingId": booking_id,
+        "offenderNo": offender_no,
+        "agencyLocationId": prison,
+        "sentenceDetail": { "homeDetentionCurfewEligibilityDate": "2011-11-07",
+                            "sentenceStartDate": "2009-02-08"
+                          }
+    }]
+  }
+
+  before do
+    signin_spo_user
+
+    stub_request(:post, "#{stub_auth_host}/auth/oauth/token").
+      with(query: { grant_type: 'client_credentials' }).
+        to_return(status: 200, body: {}.to_json)
+
+    stub_offender(offender_no, booking_number: booking_id)
+
+    stub_offenders_for_prison("LEI", offenders, bookings)
+
+    stub_request(:post, "#{stub_api_host}/movements/offenders?latestOnly=false&movementTypes=TRN").
+      with(body: [offender_no].to_json).
+        to_return(status: 200, body: [{ offenderNo: offender_no, toAgency: prison }].to_json)
+
+    stub_const("Net::IMAP", MockIMAP)
+
+    stub_const("Mail", MockMailMessage)
+
+    ENV['DELIUS_EMAIL_FOLDER'] = 'delius_import_feature'
+    ENV['DELIUS_XLSX_PASSWORD'] = 'secret'
+
+    create(:team, code: 'A')
+  end
+
+  it "imports the Delius spreadsheet and creates case information" do
```

Comment:
Is there a way to flip this test around so it checks for presence rather than absence of content?

I've found when I use `not_to have_content` in my assertions, they'll sometimes misleadingly pass - if I hit an error page, for example.

Reply:
Yeah, were now testing that its on the correct page and that the offender appears in the make allocations page.